app/services/flow/flows/provider_merits.rb

```diff
@@ -9,7 +9,7 @@ class ProviderMerits < FlowSteps
         },
         respondents: {
           path: ->(application) { urls.providers_legal_aid_application_respondent_path(application) },
-          forward: :client_received_legal_helps,
+          forward: ->(application) { application.proceeding_type?('Domestic Abuse') ? :statement_of_cases : :client_received_legal_helps },
```

Comment:
Testing the value of a string like that doesn't feel robust.
Looking at the data, it looks like there is also a `ccms_matter_code` column so I suggest we use that instead.
And to write it in a constant. In the `ProceedingType` class would make sense:
```ruby
DOMESTIC_ABUSE_CCMS_MATTER_CODE = 'MINJN'.freeze
```